ModelFarm/src/main/java/io/deephaven/modelfarm/ModelFarmOnDemand.java

```diff
@@ -20,11 +20,13 @@
  * A ModelFarm implementation for evaluating a model upon request, retrieving a snapshot of data for all keys under a
  * single {@link UpdateGraphProcessor} lock.
  *
- * @param <KEYTYPE> The type of the keys (e.g. {@link io.deephaven.modelfarm.fitterfarm.FitScope}).
+ * TODO None of the following classes exist any more - should we remove this too?
```

Comment:
@chipkent I believe this is best directed at you, along with the other ModelFarm classes. Should these classes still exist, or should we just remove these references?

Reply:
For these, just go very generic.  This is very core code that some user is going to want, but we left the finance specific part in DHE.  This is why the refs are gone.

```
@param <KEYTYPE> The type of keys.
@param <DATATYPE> The type of data.
@param <ROWDATAMANAGERTYPE> The type of RowDataManager.
```

Reply:
Use the doc strings in my comment above.  If we have a `RowDataManager` class, add a link.  If not, just change it to "row data manager".